### DISCUSSION TITLE
Remove Borg completly and reinstall it from scratch

### DISCUSSION THREAD

**Discussion by davidovic999** [mikeberger/borg_calendar#199] [General]
Hi Mike,
How do I uninstall and reinstall Borg again?
I've uninstalled it with an uninstaller program and manually deleted all the remaining files and registry entries, but on reinstallation, the program still re-displays the settings I'd made before!
How can I completely remove Borg from my computer and reinstall it from scratch?

Thanks in advance,

**Comment by mikeberger** [comment]
The uninstaller should remove the program files.

The uninstaller doesn't remove the registry items in HKEY_CURRENT_USER\Software\JavaSoft\Prefs\net\sf\borg\common\util, so remove these. These are the program's stored preferences.
You also would need to remove the database. the database folder is shown in the database options. The default is a folder called .borg_db in your user folder.

I assume you are on windows since you mention registry. On linux, the way to delete the preferences is different.

**Comment by mikeberger** [comment]
You originally posted about language settings. 

You can remove HKEY_CURRENT_USER\Software\JavaSoft\Prefs\net\sf\borg\common\util\language and HKEY_CURRENT_USER\Software\JavaSoft\Prefs\net\sf\borg\common\util\country and restart the program to reset the language.

Once you change the language, the entries in the locale menu may be displayed in a language that you can't read or in fonts that your system can't display.

**Comment by davidovic999** [comment]
Thank you for your support!
